### DISCUSSION TITLE
테이블 속성명 너무 길어지는 문제 이렇게 해결해보는게 어떨까요?

### DISCUSSION THREAD

**Discussion by kkkkimtaehyeon** [nhnacademy-be8-plzbuybook/bookstore-shoppingmall#125] [General]
![image](https://github.com/user-attachments/assets/dad5a718-0d89-43c2-bd16-097e94ee33da)

이렇게 컨벤션 정하면 정확하기는한데 관계테이블의 경우 속성명이 길어질 수가 있어서
너무 길다 싶으면 or 아예 전체 다 외래키는 테이블 줄임말 + 속성명으로 하는게 어때요?

예: opw_order_product_id

**Comment by Gksrufdk09** [comment]
오오 좋은것같아요 ! 줄여도 좀 긴 느낌이있긴 한데,,,
